schema.json

```diff
@@ -202,6 +202,80 @@
             },
             "type": "object"
         },
+        "persist": {
+            "anyOf": [
+                {
+                    "description": "Deprecated, use object representation instead.",
+                    "type": "string"
+                },
+                {
+                    "description": "Deprecated, use object representation instead.",
+                    "items": {
+                        "$ref": "#/definitions/stringOrArrayOfStrings"
+                    },
+                    "minItems": 1,
+                    "type": "array"
+                },
+                {
+                    "minItems": 1,
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "additionalProperties": false,
+                        "required": [
+                            "name",
+                            "type"
+                        ],
+                        "if": {
+                            "properties": {
+                                "type": {
+                                    "const": "file"
+                                }
+                            }
+                        },
+                        "then": {
+                            "required": [
+                                "encoding",
+                                "content"
+                            ]
+                        },
+                        "properties": {
+                            "name": {
+                                "type": "string"
+                            },
+                            "target": {
+                                "type": "string"
+                            },
+                            "type": {
+                                "enum": [
+                                    "file",
+                                    "directory",
+                                    "folder"
```

Comment:
Why both folder and directory? Is it just alias or some actual differences?

Reply:
Yeah, might drop it. Only `directory` should be fine.